be/src/runtime/exec_env.cpp

```diff
@@ -211,8 +211,8 @@ Status ExecEnv::init(const std::vector<StorePath>& store_paths, bool as_cn) {
 
     // The _load_rpc_pool now handles routine load RPC and table function RPC.
     RETURN_IF_ERROR(ThreadPoolBuilder("load_rpc") // thread pool for load rpc
-                            .set_min_threads(10)
-                            .set_max_threads(1000)
+                            .set_min_threads(1)
+                            .set_max_threads(1)
```

Comment:
Why do you change the number of threads to 1? And why it can fix this issue after this change? Will it cause other issues?

Reply:
This change is only used for test. I'll remove it.